app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,245 @@
+    <string name="alarm_time_title_30_minutes_before">30 minut przed</string>
+    <string name="alarm_time_title_45_minutes_before">45 minut przed</string>
+    <string name="alarm_time_title_60_minutes_before">60 minut przed</string>
+
+    <!-- Session list item -->
+    <string name="session_list_item_alarm_time_zero_minutes_content_description">Przypomnienie: w momencie rozpoczęcia</string>
+    <string name="session_list_item_alarm_time_minutes_content_description">Przypomnienie: <xliff:g example="5" id="minutes">%d</xliff:g> minut przed</string>
+    <string name="session_list_item_alarm_time_content_description">Czas przypomnienia: <xliff:g example="16:20" id="alarmTime">%s</xliff:g></string>
+    <string name="session_list_item_duration_text"><xliff:g example="45" id="duration">%d</xliff:g> min.</string>
+    <string name="session_list_item_duration_content_description">Czas trwania: <xliff:g example="45" id="duration">%d</xliff:g> minut</string>
+    <string name="session_list_item_favored_content_description">polubione</string>
+    <string name="session_list_item_not_favored_content_description">nie polubione</string>
+    <string name="session_list_item_room_content_description">Sala: <xliff:g example="Main hall" id="room">%s</xliff:g></string>
+    <string name="session_list_item_start_time_content_description">Rozpoczęcie <xliff:g example="16:30" id="startTime">%s</xliff:g></string>
+    <string name="session_list_item_subtitle_content_description">Napisy: <xliff:g example="News from Berlin" id="subtitle">%s</xliff:g></string>
+    <string name="session_list_item_track_content_description">Ścieżka: <xliff:g example="Security" id="trackName">%s</xliff:g></string>
+    <string name="session_list_item_language_content_description">Język: <xliff:g example="English" id="language">%s</xliff:g></string>
+    <string name="session_list_item_language_portuguese_content_description">portugalski</string>
+    <string name="session_list_item_language_german_content_description">niemiecki</string>
+    <string name="session_list_item_language_english_content_description">angielski</string>
+    <string name="session_list_item_language_unknown_content_description">Nieznany język</string>
+    <string name="session_list_item_language_removed_content_description">Usunięto informację o języku</string>
+    <string name="session_list_item_no_video_content_description">Bez nagrania</string>
+    <string name="session_list_item_video_content_description">Z nagraniem</string>
+    <string name="session_list_item_without_video_content_description">Bez nagrania</string>
+    <plurals name="session_list_item_speakers_content_description">
+        <item quantity="zero">Brak prelegentów</item>
+        <item quantity="one">Prelegent: <xliff:g example="Jane Doe" id="name">%s</xliff:g></item>
+        <item quantity="other">Prelegenci: <xliff:g example="Jane Doe, John Doe" id="names">%s</xliff:g></item>
+    </plurals>
```

Comment:
Android Studio (Lint) points out the following for plurals in Polish:

> For locale "pl" (Polish) the following quantities should also be defined: few, many

![plurals](https://user-images.githubusercontent.com/144518/145777800-1a79cf13-b29b-4504-8d68-aa57e8b81850.png)

Can you please check and probably complete the plural occurrences in this file if the suggestion fit to the language. Please be critical - Lint might not know Polish as good as you do.

Please keep `zero` here because we want another text.

Reply:
OK, I added 'few' and 'many'.

Reply:
Can you check the other plurals in the file, too please?

Reply:
It might not make sense to put few/many in the particular context. It is hard for me to tell with the Polish translation. The initial reason for plurals is the different words/word endings.

Reply:
OK, few and many fitted also here: `schedule_changes_dialog_number_of_sessions`, so I did extend it by these two cases.